pykrige/ok.py

```diff
@@ -273,7 +310,8 @@ def __init__(self, x, y, z, variogram_model='linear', variogram_parameters=None,
                 print("Using Custom Variogram Model")
             else:
                 print("Using '%s' Variogram Model" % self.variogram_model)
-                print("Sill:", self.variogram_model_parameters[0])
+                print("Partial Sill:", self.variogram_model_parameters[0])
+                print("Full Sill:", self.variogram_model_parameters[0] + self.variogram_model_parameters[2])

```

Comment:
Partial sill is just the full sill minus the nugget -- the full sill is the make value the variogram reaches, and the partial sill is how much further above the nugget (y-intercept) that variogram goes